mayastor/src/bin/main.rs

```diff
@@ -44,10 +49,21 @@ fn main() -> Result<(), std::io::Error> {
         if uring_supported { "yes" } else { "no" }
     );
     info!("free_pages: {} nr_pages: {}", free_pages, nr_pages);
-    let env = MayastorEnvironment::new(args);
-    env.start(|| {
-        info!("Mayastor started {} ...", '\u{1F680}');
```

Comment:
Is there a replacement for this message?

Reply:
Ah good point